src/config.js

```diff
@@ -178,3 +181,15 @@ export const KEDRO_VIZ_PREVIEW_DATASETS_DOCS_URL = `${KEDRO_VIZ_DOCS_URL}preview
 export const KEDRO_VIZ_PUBLISH_AWS_DOCS_URL = `${KEDRO_VIZ_DOCS_URL}publish_and_share_kedro_viz_on_aws.html#set-up-endpoint`;
 export const KEDRO_VIZ_PUBLISH_AZURE_DOCS_URL = `${KEDRO_VIZ_DOCS_URL}publish_and_share_kedro_viz_on_azure.html#set-up-endpoint`;
 export const KEDRO_VIZ_PUBLISH_GCP_DOCS_URL = `${KEDRO_VIZ_DOCS_URL}publish_and_share_kedro_viz_on_gcp.html#set-up-endpoint`;
+
+export const queryParamsToRetain = [
+  params.types,
+  params.tags,
```

Comment:
Is this array to retain the default query params ? If so we should have `params.pipeline` and should not have `params.tags` right ?

Reply:
I missed `params.pipeline` and added but `params.tags` will also be part of query params to retain when user clicks on reset view. Also in existing behaviour, on reset view click we do not clear any tag if already applied.

Reply:
I guess that might be a design question. From what I understand **Reset view** should take me back to an initial load. Our initial load (which does not have prior session) should only have - `types=task,data&pid=__default__&expandAllPipelines=false`. But I would like to hear from @rashidakanchwala and @Huongg

Reply:
any comment on this ? @rashidakanchwala